lib/pure/strutils.nim

```diff
@@ -130,6 +145,24 @@ func isAlphaAscii*(c: char): bool {.rtl, extern: "nsuIsAlphaAsciiChar".} =
     doAssert isAlphaAscii('8') == false
   return c in Letters
 
+func isVowelAscii*(c: char): bool {.rtl, extern: "nsuIsVowelAsciiChar".} =
```

Comment:
what's the point? `c in Vowels` is good enough

Reply:
I just added them because functions for all other, like `isAlphaAscii` exist.
It would 100% make it easier if they didn't exist, I simply added them to make it more consistent with some of the other functions like this.

Reply:
I think `isAlphaAscii` + friends only exist for historical reasons, consistency is less important than avoid bloat here